examples/common/lib/utils.sh

```diff
@@ -139,6 +139,31 @@ function wait_for_healthy_shard() {
 	wait_for_shard_vreplication_engine "${keyspace}" "${shard}"
 }
 
+# Stop the specified vtadmin binary name using the provided PID file.
+# Example:
+#  stop_vtadmin_process "vtadmin-web" "$VTDATAROOT/tmp/vtadmin-web.pid"
+function stop_vtadmin_process() {
```

Comment:
I would suggest changing this function name to `stop_binary_pid` and reuse for other things like `vtctld-down.sh`, `vtgate-down.sh`, `vtorc-down.dh` and probably others.